docs/troubleshooting.md

```diff
@@ -0,0 +1,34 @@
+Troubleshooting
+---------------
+
+Here are some errors you might encounter while developing the extension, and
```

Comment:
This is fine, although I'll note that these are all conditions that could be checked quite easily in the code that handles the injection. So we could potentially display a useful error page to developers (via the existing help page system) rather than having errors hidden away in the console.